homeassistant/components/velbus/manifest.json

```diff
@@ -2,8 +2,9 @@
   "domain": "velbus",
   "name": "Velbus",
   "documentation": "https://www.home-assistant.io/integrations/velbus",
-  "requirements": ["python-velbus==2.1.2"],
+  "requirements": ["velbus-aio==2021.8.7"],
   "config_flow": true,
   "codeowners": ["@Cereal2nd", "@brefra"],
-  "iot_class": "local_push"
+  "iot_class": "local_push",
+  "quality_scale": "gold"
```

Comment:
Tests are missing for gold level. Please change quality_scale in a separate PR after all requirements are done.